pkg/watchdog/init.sh

```diff
@@ -4,6 +4,17 @@ USE_HW_WATCHDOG=1
 DEFAULT_WATCHDOG_CHANGE_TIME=300
 WATCHDOG_CHANGE_TIME=$(</proc/cmdline grep -o '\bchange=[^ ]*' | cut -d = -f 2)
 
+adjust_parent_oom_score() {
+  SELF_OOM_SCORE_ADJ=$(cat /proc/$$/oom_score_adj)
+  PARENT_OOM_SCORE_ADJ=$SELF_OOM_SCORE_ADJ
+  if [ -n "$SELF_OOM_SCORE_ADJ" ] && [ "$SELF_OOM_SCORE_ADJ" -eq "$SELF_OOM_SCORE_ADJ" ] 2>/dev/null; then
```

Comment:
What is the eq check? That it is a number? Could you get something other than a number (or empty string) from the cat of that file?

Reply:
Good point, looks like noop